README.md

```diff
@@ -1,6 +1,6 @@
 # Clippy
 
-[![Clippy Test](https://github.com/rust-lang/rust-clippy/workflows/Clippy%20Test%20(bors)/badge.svg?branch=auto&event=push)](https://github.com/rust-lang/rust-clippy/actions?query=workflow%3A%22Clippy+Test+(bors)%22+event%3Apush+branch%3Aauto)
+[![Clippy Test](https://github.com/rust-lang/rust-clippy/workflows/Clippy%20Test%20(merge%20queue)/badge.svg?branch=auto&event=push)](https://github.com/rust-lang/rust-clippy/actions?query=workflow%3A%22Clippy+Test+(bors)%22+event%3Apush+branch%3Aauto)
```

Comment:
No, we don't want to run anything on master. Actually, with a merge queue, I think that the badge is redundant, lol. By definition, master has to be green, otherwise there would be a bug in the merge queue.. :)

Reply:
> (The same was true for bors, btw)

Yeah, I know. I just like green badges :D

Then I'd say, let's just remove the badge, as I don't think we can generate any badge that makes sense with merge queues?

Reply:
I suppose that we can simply avoid the `?branch` and `event` parametrs, and just use the name of the CI job that runs on merge queues, that should work.

Reply:
Pushed a fix to the badge URLs.